db/migrate/20170112194141_change_name_to_friendly_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeNameToFriendlyName < ActiveRecord::Migration
+  def change
+    rename_column :service_providers, :name, :friendly_name
```

Comment:
How are we planning on running this migration? Are we going to deploy code that aliases the column name first? +1/-1 compatibility? Or are we just going to take downtime while we migrate the DB?